src/dawn/engine_dawn.cc

```diff
@@ -36,7 +38,48 @@ namespace {
 
 // The minimum multiple row pitch observed on Dawn on Metal.  Increase this
 // as needed for other Dawn backends.
-const uint32_t kMinimumImageRowPitch = 256;
+static const uint32_t kMinimumImageRowPitch = 256;
+static const float kLodMin = 0.0;
+static const float kLodMax = 1000.0;
+static const uint32_t kMaxColorAttachments = 4u;
+static const uint32_t kMaxVertexInputs = 16u;
+static const uint32_t kMaxVertexAttributes = 16u;
+
+// This structure is a container for a few variables that are created during
+// CreateRenderPipelineDescriptor and CreateRenderPassDescriptor and we want to
+// make sure they don't go out of scope before we are done with them
```

Comment:
The comment about the lifetime of the values is insightful.  Lifecycle info is an important aspect that's often overlooked, and the source of bugs.